packages/demos/stories/TableColumnsDragDrop.tsx

```diff
@@ -0,0 +1,286 @@
+/**
+ * SPDX-FileCopyrightText: © 2020 Liferay, Inc. <https://liferay.com>
+ * SPDX-License-Identifier: BSD-3-Clause
+ */
+
+import ClayTable from '@clayui/table';
+import classNames from 'classnames';
+import React from 'react';
+import {
+	DropTargetMonitor,
+	XYCoord,
+	useDrag,
+	useDragLayer,
+	useDrop,
+} from 'react-dnd';
+
+const TABLE_COLUMN_DND_TYPE = 'tableColumn';
+
+const initialTableColumns = [
+	{
+		data: ['Red, white & blue', 'Tricolore', 'Stars & stripes'],
+		id: 'flag',
+		title: 'Flag',
+	},
+	{
+		data: ['Europe', 'Europe', 'North America'],
+		id: 'region',
+		title: 'Region',
+	},
+	{
+		data: ['Croatia', 'Italy', 'USA'],
+		id: 'country',
+		title: 'Country',
+	},
+	{
+		data: [
+			'Lorem ipsum dolor, sit amet consectetur adipisicing elit.',
+			'Lorem ipsum dolor, sit amet consectetur adipisicing elit.',
+			'Lorem ipsum dolor, sit amet consectetur adipisicing elit.',
+		],
+		id: 'description',
+		title: 'Description',
+	},
+];
+
+const dragLayerStyles: React.CSSProperties = {
+	height: '100%',
+	left: 0,
+	pointerEvents: 'none',
+	position: 'fixed',
+	top: 0,
+	width: '100%',
+	zIndex: 100,
```

Comment:
Where is this zIndex from? I think we should probably grab these from clay-css if possible

Reply:
The `100`was in the example, now I grabbed `1060` from [_globals-z-index.scss#L6](https://github.com/liferay/clay/blob/master/packages/clay-css/src/scss/variables/_globals-z-index.scss#L6). Not sure if we can somehow access SCSS variables from JSX.

Reply:
No worries about grabbing a variable, as long as we are in sync with our css definition we should be good